### PR TITLE
Add Kalalokki

### PR DESCRIPTION
**Pull request by Spandamn**

### CODE REVIEW

**Comment by Spandamn** [comment]
??? ```this.forceSwitch``` should work with 4 args?

**Comment by Spandamn** [comment]
https://github.com/smogon/pokemon-showdown/blob/master/data/scripts.ts#L1070

**Comment by ghost** [comment]
It Does that because https://github.com/smogon/pokemon-showdown/blob/deeed6c78951b98679a99b81346a8994be7183db/sim/battle.ts#L3063

**Comment by Spandamn** [comment]
@xInstruct what should i assign to damage

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -1198,6 +1198,35 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Hurricane', target);
+		},
+		onAfterHit(target, source, move) {
+			let count = 0;
+			while (source.side.foe.active[0] && source.side.foe.active[0] !== target && count++ < 2) {
+				this.forceSwitch(undefined, source.side.foe.active, source, move, move);
```
Comment (ghost) [inline data/mods/ssb/moves.ts]: reL t
```suggestion
				this.forceSwitch(null, source.side.foe.active, source, move, move);
```

**Comment by ghost** [comment]
fug

well i try sometime tomorrow. if you want you can just take litt and dragonwhale back if u want

**Comment by Spandamn** [comment]
nah not yet

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -1198,6 +1198,45 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+		onHit(target, source, move) {
+			target.side.addSideCondition('gaelstrom');
+			target.side.sideConditions['gaelstrom'].firstTarget = target;
+			target.side.sideConditions['gaelstrom'].sourceMove = move;
+		},
+		condition: {
+			onSwitchIn(pokemon) {
+				if (!this.effectData.count) this.effectData.count = 1;
+				if (this.effectData.firstTarget !== pokemon && this.effectData.count++ < 3) {
+					this.forceSwitch([0], pokemon.side.active, this.effectData.source, this.effectData.sourceMove, this.effectData.sourceMove);
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion
					pokemon.forceSwitchFlag = true;
```

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -1198,6 +1198,45 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+				}
+				if (this.effectData.count >= 3 || !this.effectData.source || this.effectData.source.fainted || this.effectData.source.hp <= 0) pokemon.side.removeSideCondition('gaelstrom');
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion
				if (this.effectData.count >= 3 || !this.effectData.source || this.effectData.source.fainted || this.effectData.source.hp <= 0) {
					pokemon.side.removeSideCondition('gaelstrom');
				}
```

File: data/mods/ssb/moves.ts [hunk 4]
```diff
@@ -1198,6 +1198,47 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+			target.side.sideConditions['gaelstrom'].firstTarget = target;
+			target.side.sideConditions['gaelstrom'].sourceMove = move;
+		},
+		condition: {
+			onSwitchIn(pokemon) {
+				if (!this.effectData.count) this.effectData.count = 1;
+				if (this.effectData.firstTarget !== pokemon && this.effectData.count++ < 3) {
+					pokemon.forceSwitchFlag = true;
+				}
+				if (this.effectData.count >= 3 || !this.effectData.source || this.effectData.source.fainted || this.effectData.source.hp <= 0) {
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion
				if (this.effectData.count >= 3 || !this.effectData.source ||
					this.effectData.source.fainted || this.effectData.source.hp <= 0) {
```

**Pull request merged**